pywr/parameters/_parameters.pyx

```diff
@@ -937,6 +937,12 @@ cdef class RbfProfileParameter(Parameter):
         self._upper_bounds = np.ones(self.double_size)*upper_bounds
 
         if self.variable_days_of_year_range > 0:
+            if np.any(np.diff(self.days_of_year) <= 2*self.variable_days_of_year_range):
+                raise ValueError(f"The days of the year are too close together for the given "
+                                 f"`variable_days_of_year_range`. This could cause the optimised days"
+                                 f"of the year to overlap and become out of order.  Either increasing the"
```

Comment:
should be 'increase' rather than 'increasing'

Reply:
Done.